include/FreeRTOS_IP.h

```diff
@@ -126,6 +126,15 @@
     #define ipICMP_NEIGHBOR_SOLICITATION_IPv6          ( ( uint8_t ) 135U )
     #define ipICMP_NEIGHBOR_ADVERTISEMENT_IPv6         ( ( uint8_t ) 136U )
 
+    #define ipIPv6_EXT_HEADER_HOP_BY_HOP               0U
```

Comment:
NIT: Can we arrange them in ascending order? ( 0 ->43->44->50->51->60).
I am fine with this as well as this is arranged in the order the headers should be received.

Reply:
I declared the options them in the order in which they appear in an extension header. I can see that in the new function `xGetExtensionOrder()`.